pkgs/tools/filesystems/darling-dmg/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchFromGitHub, lib, cmake, fuse, zlib, bzip2, openssl, libxml2, icu } :
+
+stdenv.mkDerivation rec {
+  name = "darling-dmg";
```

Comment:
Is there a check list for these kind of things or does everyone have to learn the nuances of these hassles?